WreckEngine/Source.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
```

Comment:
NGL, not a fan of this. Not only should this file not be included anywhere else or it'll mess things up, it also obfuscates what's going on in Source.cpp. I realize this is primarily to avoid having to move any declarations above the includes, but there aren't thaaat many.